### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Feliverse**
**Project requirements**
API to base the development of the webapp on.:
https://financialmodelingprep.com

**Design**

You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size and weight.
Layout: composition and space between elements.
Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).

**Interactions**

**Home page**

When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
Along with the category name, you will display some numeric values per category that come from the API.
When the user clicks (or taps) on a category item, the application navigates to the details page.
Details page

In the details page, the webapp retrieves data from the API to show detailed data in the category.
When the user clicks on the "Back" button (<), the user navigates to the home page.

**Testing requirements**

Create unit tests for pure functions in your Redux code.
Create integration tests for your application using the React Testing Library.
You may need to mock the access to the API, so that your tests don't send actual requests.
You may need to mock the connection to the Redux Store.
**Technical requirements**

The project is a single-page application (SPA) built with React and Redux.
The data retrieved from the API should be stored in the Redux store.
You should filter the data that you retrieve from the API using a Filter stateless component.
Every page (the main page and the pages for each item) should have a unique route within the SPA.
The project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/).
Add a link to the online version of your app to your README file.
Project documentation
Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

a description of the project.
a demo of the project features.
you should also highlight some interesting piece of code or something you built that you are very proud of.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,223 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project bring me a star ⭐️ at the Github repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (VICTOR-CHUKWU) [inline README.md:199]: - [x] kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    if (!efts.length) {
+      dispatch(getEFTs());
+    }
+  }, [cryptos.length, dispatch, efts.length, forex.length, stocks.length]);
+
+  return (
+    <section className="home-page">
+      <div className="top-bar">
```
Comment (VICTOR-CHUKWU) [inline src/components/Home.js:39]: - [ ] good job @Feliverse  in this page a user should be able to filter the crypto list either by name or price, kindly add a filter or search bar kind of component where a user can search for a particular crypto :+1:

File: src/__tests__/__snapshots__/snapshots.test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        </Router>
+      </Provider>,
+    ).toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('matches EFTs snapshot', () => {
+    const tree = TestRenderer.create(
+      <Provider store={store}>
```
Comment (VICTOR-CHUKWU) [inline src/__tests__/__snapshots__/snapshots.test.js:40]: - [x] some of your tests are failing, kindly fix this
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Feliverse , 
## STATUS: CHANGES REQUIRED :recycle: 
Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Feliverse) [reply README.md:199]: @VICTOR-CHUKWU , I will create a CC licence in a while

File: src/components/Home.js (hunk 2 above)
Comment (Feliverse) [reply src/components/Home.js:39]: @VICTOR-CHUKWU thank you for the suggestion, I will consider it implemented the next week, for now, it is not a requirement for the capstone.

File: src/__tests__/__snapshots__/snapshots.test.js (hunk 3 above)
Comment (Feliverse) [reply src/__tests__/__snapshots__/snapshots.test.js:40]: @VICTOR-CHUKWU, 
tests fixed, 
Thank you for your review

**Review by brytebee — COMMENTED** [review]
Hello, @Feliverse 👋 ,

------

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

------

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**